bytestring.cabal

```diff
@@ -1,5 +1,5 @@
 Name:                bytestring
-Version:             0.10.10.1
+Version:             0.10.12.0
```

Comment:
Why is it `0.10.12.0` and not `0.10.11.0`?

Reply:
I simply stuck with the existing scheme for the `0.10.*` series to use only even numbers for the minor version components.

I don't know the reason for this scheme though. @dcoutts, could you help?